apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureActivity.tsx

```diff
@@ -188,12 +188,7 @@ const InfrastructureActivity = () => {
       </ScaffoldContainer>
       <ScaffoldContainer className="sticky top-0 py-6 border-b bg-background z-10">
         <div className="flex items-center gap-x-4">
-          {/* {readReplicasEnabled && (
-            <DatabaseSelector
-              selectedDatabaseId={selectedDatabaseId}
-              onChangeDatabaseId={setSelectedDatabaseId}
-            />
-          )} */}
+          {/* {readReplicasEnabled && <DatabaseSelector />} */}
```

Comment:
Still want this commented out?

Reply:
Yup! activity metrics for replicas are still not done, i'll leave a comment actually